src/Command/stubs/service.stub

```diff
@@ -49,7 +49,7 @@ class DummyClass
     {
         $dummyname = $this->dummynameRepository->create($data);
 
-        $this->dispatcher->fire(new DummyModelWasCreated($dummyname));
+        $this->dispatcher->dispatch(new DummyModelWasCreated($dummyname));
```

Comment:
Since laravel 5 fire has been deprecated in favor of dispatch, so updating it so generated services remains working properly